operator/v1/types_network.go

```diff
@@ -123,6 +123,34 @@ type NetworkMigration struct {
 	// changing the MTU for the default network will be rejected.
 	// +optional
 	MTU *MTUMigration `json:"mtu,omitempty"`
+
+	// features contains the features migration configuration. Set this to migrate
+	// feature configuration when changing the cluster default network provider.
```

Comment:
Do we support feature migration in both directions? Or is this one-way? Should clarify if the latter.

Reply:
We support in both directions.